src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Shooter;
+
+
+public class ShootCommand extends Command {
+
+    private final Shooter shooter;
+    private final XboxController controller;
```

Comment:
The controller button should not be defined inside of the class and instead inside of robot container.